napari/_qt/layer_controls/qt_layer_controls_container.py

```diff
@@ -148,7 +165,6 @@ def _remove(self, event):
         layer = event.value
         controls = self.widgets[layer]
         self.removeWidget(controls)
-        # controls.close()
```

Comment:
I have a passion for deleting commented code that lacks any explanatory comments.